<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrbitMainWindow</class>
 <widget class="QMainWindow" name="OrbitMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orbit Profiler</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit[accessibleName=&quot;filter&quot;] {
	background-image: url(:/actions/search_small_offset);
	background-position: left center;
	background-repeat: no-repeat;
	height: 22px;
	padding-left: 20px;
	padding-bottom: 1px;
}

QToolTip {
	color: #ffffff;
	background-color: #2a82da;
	border: 1px
	solid white;
}

QPushButton:disabled {
	color: gray;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="MainTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="HomeTab">
        <attribute name="title">
         <string>Home</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="1">
          <widget class="QSplitter" name="HomeHorizontalSplitter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QSplitter" name="HomeVerticalSplitter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="ProcessLauncherWidget" name="ProcessesList" native="true"/>
            <widget class="OrbitDataViewPanel" name="SessionList" native="true"/>
           </widget>
           <widget class="OrbitDataViewPanel" name="ModulesList" native="true"/>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="CaptureTab">
        <attribute name="title">
         <string>Capture</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolBar" name="capture_toolbar">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <addaction name="actionToggle_Capture"/>
           <addaction name="actionClear_Capture"/>
           <addaction name="actionOpen_Capture"/>
           <addaction name="actionSave_Capture"/>
           <addaction name="actionHelp"/>
           <addaction name="actionIntrospection"/>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="filterPanel">
           <property name="accessibleName">
            <string/>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="filterTracks">
              <property name="accessibleName">
               <string>filter</string>
              </property>
              <property name="styleSheet">
               <string notr="true">background-image: url(:/actions/filter_list_small_offset);</string>
              </property>
              <property name="placeholderText">
               <string>Filter Tracks</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="filterFunctions">
              <property name="accessibleName">
               <string>filter</string>
              </property>
              <property name="placeholderText">
               <string>Filter Functions</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../icons/orbiticons.qrc">:/actions/access_time</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="timerLabel">
              <property name="text">
               <string>1s</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="OrbitGLWidget" name="CaptureGLWidget"/>
         </item>
        </layout>
        <zorder>CaptureGLWidget</zorder>
        <zorder>capture_toolbar</zorder>
        <zorder>filterPanel</zorder>
       </widget>
      </widget>
      <widget class="QTabWidget" name="RightTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="FunctionsTab">
        <attribute name="title">
         <string>Functions</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="OrbitDataViewPanel" name="FunctionsList" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="CallStackTab">
        <attribute name="title">
         <string>Callstack</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="OrbitDataViewPanel" name="CallStackView" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="liveTab">
        <attribute name="title">
         <string>Live</string>
        </attribute>
        <layout class="QGridLayout" name="liveGridLayout">
         <item row="0" column="0">
          <widget class="OrbitLiveFunctions" name="liveFunctions" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="samplingTab">
        <attribute name="title">
         <string>Sampling</string>
        </attribute>
        <layout class="QGridLayout" name="samplingGridLayout">
         <item row="0" column="0">
          <widget class="OrbitSamplingReport" name="samplingReport" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="topDownTab">
        <attribute name="title">
         <string>Top-Down</string>
        </attribute>
        <layout class="QGridLayout" name="topDownGridLayout">
         <item row="0" column="0">
          <widget class="CallTreeWidget" name="topDownWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="bottomUpTab">
        <attribute name="title">
         <string>Bottom-Up</string>
        </attribute>
        <layout class="QGridLayout" name="bottomUpGridLayout">
         <item row="0" column="0">
          <widget class="CallTreeWidget" name="bottomUpWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="selectionSamplingTab">
        <attribute name="title">
         <string>Sampling (selection)</string>
        </attribute>
        <layout class="QGridLayout" name="selectionGridLayout">
         <item row="0" column="0">
          <widget class="OrbitSamplingReport" name="selectionReport" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="selectionTopDownTab">
        <attribute name="title">
         <string>Top-Down (selection)</string>
        </attribute>
        <layout class="QGridLayout" name="selectionTopDownGridLayout">
         <item row="0" column="0">
          <widget class="CallTreeWidget" name="selectionTopDownWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="selectionBottomUpTab">
        <attribute name="title">
         <string>Bottom-Up (selection)</string>
        </attribute>
        <layout class="QGridLayout" name="selectionBottomUpGridLayout">
         <item row="0" column="0">
          <widget class="CallTreeWidget" name="selectionBottomUpWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="CodeTab">
        <attribute name="title">
         <string>Code</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="OrbitCodeEditor" name="CodeTextEdit">
           <property name="accessibleName">
            <string>CodeTextEdit</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QWidget" name="FileMappingWidget" native="true">
           <layout class="QGridLayout" name="gridLayout_9">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QPushButton" name="SaveFileMapping">
              <property name="text">
               <string>Save File Mapping</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="OrbitCodeEditor" name="FileMappingTextEdit">
              <property name="accessibleName">
               <string>FileMappingTextEdit</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tracepointsTab">
        <attribute name="title">
         <string>Tracepoints</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="OrbitDataViewPanel" name="TracepointsList" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="debugTab">
        <attribute name="title">
         <string>Debug</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="OrbitGLWidget" name="debugOpenGLWidget"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Capture"/>
    <addaction name="actionSave_Capture"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Preset"/>
    <addaction name="actionSave_Preset_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFeedback">
    <property name="title">
     <string>Feedback</string>
    </property>
    <addaction name="actionReport_Missing_Feature"/>
    <addaction name="actionReport_Bug"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <widget class="QMenu" name="menuCrash">
     <property name="title">
      <string>Crash Test</string>
     </property>
     <addaction name="actionCheckFalse"/>
     <addaction name="actionNullPointerDereference"/>
     <addaction name="actionStackOverflow"/>
     <addaction name="actionServiceCheckFalse"/>
     <addaction name="actionServiceNullPointerDereference"/>
     <addaction name="actionServiceStackOverflow"/>
    </widget>
    <addaction name="menuCrash"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDebug"/>
   <addaction name="menuHelp"/>
   <addaction name="menuFeedback"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReport_Missing_Feature">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/feedback</normaloff>:/actions/feedback</iconset>
   </property>
   <property name="text">
    <string>Report Missing Feature</string>
   </property>
  </action>
  <action name="actionReport_Bug">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/bug_report</normaloff>:/actions/bug_report</iconset>
   </property>
   <property name="text">
    <string>Report Bug</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/info</normaloff>:/actions/info</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_Preset">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/bookmarks</normaloff>:/actions/bookmarks</iconset>
   </property>
   <property name="text">
    <string>Open Preset...</string>
   </property>
  </action>
  <action name="actionToggle_Capture">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/play_arrow</normaloff>:/actions/play_arrow</iconset>
   </property>
   <property name="text">
    <string>Toggle Capture</string>
   </property>
   <property name="toolTip">
    <string>Toggle Capture</string>
   </property>
  </action>
  <action name="actionClear_Capture">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/clear</normaloff>:/actions/clear</iconset>
   </property>
   <property name="text">
    <string>Clear Capture</string>
   </property>
   <property name="toolTip">
    <string>Clear Capture</string>
   </property>
  </action>
  <action name="actionFilter_Tracks">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/filter_list</normaloff>:/actions/filter_list</iconset>
   </property>
   <property name="text">
    <string>Filter Tracks</string>
   </property>
   <property name="toolTip">
    <string>Filter Tracks</string>
   </property>
  </action>
  <action name="actionFilter_Functions">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/search</normaloff>:/actions/search</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="toolTip">
    <string>Search</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/help</normaloff>:/actions/help</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Help</string>
   </property>
  </action>
  <action name="actionIntrospection">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/info</normaloff>:/actions/info</iconset>
   </property>
   <property name="text">
    <string>Introspection</string>
   </property>
   <property name="toolTip">
    <string>Introspection</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/exit_to_app</normaloff>:/actions/exit_to_app</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionToogleDevMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dev Mode</string>
   </property>
  </action>
  <action name="actionSave_Preset_As">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/bookmark_border</normaloff>:/actions/bookmark_border</iconset>
   </property>
   <property name="text">
    <string>Save Preset As...</string>
   </property>
  </action>
  <action name="actionOpen_Capture">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/folder</normaloff>:/actions/folder</iconset>
   </property>
   <property name="text">
    <string>Open Capture...</string>
   </property>
  </action>
  <action name="actionSave_Capture">
   <property name="icon">
    <iconset resource="../icons/orbiticons.qrc">
     <normaloff>:/actions/save_alt</normaloff>:/actions/save_alt</iconset>
   </property>
   <property name="text">
    <string>Save Capture</string>
   </property>
  </action>
  <action name="actionCheckFalse">
   <property name="text">
    <string>Check False</string>
   </property>
  </action>
  <action name="actionNullPointerDereference">
   <property name="text">
    <string>Null Pointer Dereference</string>
   </property>
  </action>
  <action name="actionStackOverflow">
   <property name="text">
    <string>Stack Overflow</string>
   </property>
  </action>
  <action name="actionServiceCheckFalse">
   <property name="text">
    <string>[Service] Check False</string>
   </property>
  </action>
  <action name="actionServiceNullPointerDereference">
   <property name="text">
    <string>[Service] Null Pointer Dereference</string>
   </property>
  </action>
  <action name="actionServiceStackOverflow">
   <property name="text">
    <string>[Service] Stack Overflow</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OrbitDataViewPanel</class>
   <extends>QWidget</extends>
   <header>orbitdataviewpanel.h</header>
  </customwidget>
  <customwidget>
   <class>OrbitCodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>orbitcodeeditor.h</header>
  </customwidget>
  <customwidget>
   <class>OrbitGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>orbitglwidget.h</header>
  </customwidget>
  <customwidget>
   <class>ProcessLauncherWidget</class>
   <extends>QWidget</extends>
   <header>processlauncherwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OrbitLiveFunctions</class>
   <extends>QWidget</extends>
   <header>orbitlivefunctions.h</header>
  </customwidget>
  <customwidget>
   <class>OrbitSamplingReport</class>
   <extends>QWidget</extends>
   <header>orbitsamplingreport.h</header>
  </customwidget>
  <customwidget>
   <class>CallTreeWidget</class>
   <extends>QWidget</extends>
   <header>CallTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons/orbiticons.qrc"/>
  <include location="../images/orbitimages.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>filterFunctions</sender>
   <signal>textChanged(QString)</signal>
   <receiver>OrbitMainWindow</receiver>
   <slot>OnFilterFunctionsTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterTracks</sender>
   <signal>textChanged(QString)</signal>
   <receiver>OrbitMainWindow</receiver>
   <slot>OnFilterTracksTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnFilterTracksTextChanged(QString)</slot>
  <slot>OnFilterFunctionsTextChanged(QString)</slot>
 </slots>
</ui>
